inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -369,7 +338,7 @@
                         "type": "string"
                     },
                     "cnum": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
It is: 189 records on production have more than one CNUM, (i.e. they might refer to more than one conference). E.g.:
http://inspirehep.net/record/1405352
http://inspirehep.net/record/1420854
http://inspirehep.net/record/1424721

Ref: https://github.com/inspirehep/inspire-next/blob/master/inspirehep/dojson/current_marcxml_usage/hep-repeatable.txt#L138